projects/ion/src/lib/radio/radio.component.scss

```diff
@@ -0,0 +1,130 @@
+@import '../../styles/index.scss';
+
+.ion-input-radio {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
+
+.labelDisabled {
+  cursor: pointer;
```

Comment:
```suggestion
```

When disabled is necessary cursor pointer?

Reply:
Acho que fica bom com o not-allowed